command/server_test.go

```diff
@@ -1,5 +1,15 @@
+// +build !race
 package command
```

Comment:
The docs don't say that that's required ("Constraints may appear in any kind of source file (not just Go), but they must appear near the top of the file, preceded only by blank lines and other line comments. These rules mean that in Go files a build constraint must appear before the package clause.") and Go's own source tree has those lines at the top of the file. Obviously something's not working right though, so I'll fix that before merging.